airbyte-ci/connectors/live-tests/pyproject.toml

```
@@ -24,6 +24,7 @@ pytest-asyncio = "~=0.23.5"
 pydash = "~=7.0.7"
 docker = ">=6,<7"
 asyncclick = "^8.1.7.1"
+connection-retriever = {git = "https://github.com/airbytehq/airbyte-platform-internal.git", rev = "augustin/03-06-create_connection-retriever_tool", subdirectory = "tools/connection-retriever"}
```

This has to be changed to 
```suggestion
connection-retriever = {git = "https://github.com/airbytehq/airbyte-platform-internal.git", rev = "main", subdirectory = "tools/connection-retriever"}
```

once https://github.com/airbytehq/airbyte-platform-internal/pull/11536 is merged.